Modules/ITS/src/ITSDecodingErrorCheck.cxx

```diff
@@ -30,14 +30,31 @@ namespace o2::quality_control_modules::its
 
 Quality ITSDecodingErrorCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
 {
+  // set timer
+  if (nCycle == 0) {
+    start = std::chrono::high_resolution_clock::now();
+    nCycle++;
+  } else {
+    end = std::chrono::high_resolution_clock::now();
+    TIME = std::chrono::duration_cast<std::chrono::seconds>(end - start).count();
+  }
   std::vector<int> vDecErrorLimits = convertToArray<int>(o2::quality_control_modules::common::getFromConfig<string>(mCustomParameters, "DecLinkErrorLimits", ""));
   if (vDecErrorLimits.size() != o2::itsmft::GBTLinkDecodingStat::NErrorsDefined) {
     LOG(error) << "Incorrect vector with DecodingError limits, check .json" << ENDM;
     doFlatCheck = true;
   }
+  std::vector<float> vDecErrorLimitsRatio = convertToArray<float>(o2::quality_control_modules::common::getFromConfig<string>(mCustomParameters, "DecLinkErrorLimitsRatio", ""));
+  if (vDecErrorLimitsRatio.size() != o2::itsmft::GBTLinkDecodingStat::NErrorsDefined) {
+    LOG(error) << "Incorrect vector with DecodingError limits Ratio, check .json" << ENDM;
```

Comment:
I will merge it like this before the upcoming QC release, but here you are mixing fairlogger with infologger interface. FairLogger does not need `ENDM` at the end, as it recognizes the end of line. `ENDM` shall be used with infologger (`ILOG(Error)`).

Reply:
OK，thanks a lot! Then, should I modify it and commit again right now or next time?

Reply:
As you prefer. If you are not waiting for Nicolo's review before merging this one, I would propose that I merge this one and you do these changes in another PR, so we are not blocked by failing CI by any chance.

Reply:
ok, thanks a lot! I will do these changes next time.